datadog/resource_datadog_dashboard.go

```diff
@@ -4713,7 +4713,27 @@ func getApmLogNetworkRumSecurityQuerySchema() *schema.Schema {
 				},
 				"compute": {
 					Type:     schema.TypeMap,
-					Required: true,
+					Optional: true,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
```

Comment:
This Schema definition seems the same as the one for `multi_compute` AFAICS. Would it make sense to create a function to return it instead of duplicating it?

Reply:
We could. The schema for `compute` is actually useless because it's a `TypeMap`, but it doesn't hurt.